src/js/components/Tiles.js

```diff
@@ -195,14 +211,23 @@ export default class IndexTiles extends Component {
       }, this);
     }
 
+    let bulkOperationsContent;
+
+    if (bulkOperationsComponent) {
+      bulkOperationsContent = <BulkOperations items={result.items} component={bulkOperationsComponent}/>;
+    }
+
     return (
-      <Tiles className={classes.join(' ')} onMore={onMore}
-        flush={this.props.flush} fill={this.props.fill}
-        selectable={this.props.onSelect ? true : false}
-        selected={selectionIndex}
-        size={this.props.size}>
-        {tiles}
-      </Tiles>
+      <Box direction="row" pad={{between: 'small'}} responsive={false}>
+        <Tiles className={classes.join(' ')} onMore={onMore}
+          flush={this.props.flush} fill={this.props.fill}
+          selectable={this.props.onSelect ? true : false}
+          selected={selectionIndex}
+          size={this.props.size}>
+          {tiles}
+        </Tiles>
+        {bulkOperationsContent}
```

Comment:
I think it might be more consistent to have actions against all items be part of IndexHeader or as a Sidebar. For instance, if there are multiple actions that could be performed for all items, they could be in a sidebar on the right, just like with actions for a single item. Then responsively, that would collapse down to a single control in IndexHeader. I'll get together with Tracy today to discuss the UX for this some more.